src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -63,8 +63,8 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
         auto vulnerabilityScan =
             [&](const std::string& cnaName, const NSVulnerabilityScanner::ScanVulnerabilityCandidate& callbackData)
         {
-            // if the platforms are not empty, we need to check if the platform is in the list.
-            if (callbackData.platforms())
+            // If the platform array is not empty, we need to check if the platform is in the list.
+            if (callbackData.platforms() && callbackData.platforms()->size())
```

Comment:
This is a workarround, the fix must implemented here
- https://github.com/wazuh/wazuh/blob/4.8.0/src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVECandidates.hpp#L103-L122

Reply:
done